.github/workflows/upload-coverage-files.yml

```
@@ -0,0 +1,52 @@
+# limitations under the License.
+#
+# SPDX-License-Identifier: Apache-2.0
+
+name: Run Go Tests on Main Branch
+
+on:
+  workflow_call:
+    outputs:
+      coverage-artifact-id:
+        description: Coverage artifact id for the main branch baseline
+        value: ${{ jobs.upload-coverage-files.outputs.coverage-artifact-id }}
```

actionlint (1.7.12)

[error] property "upload-coverage-files" is not defined in object type {upload-main-coverage-files: {outputs: {coverage-artifact-id: string}}}

(expression)